fido-device-onboard.spec

```diff
@@ -13,7 +13,7 @@ Summary:        An implementation of the FIDO Device Onboard Specification writt
 
 License:        BSD 3
 URL:            %{forgeurl}
-Source:         %{url}/archive/%{commit}/%{name}-rs-%{commit}.tar.gz
+Source:         %{forgesource}
```

Comment:
You still need a tar file in that URL, I don't see a forgesource defined anywhere in the source tree using grep.

Reply:
https://fedoraproject.org/wiki/Forge-hosted_projects_packaging_automation

Reply:
For reference the docs are at: https://docs.fedoraproject.org/en-US/packaging-guidelines/SourceURL/#_using_forges_hosted_revision_control